### PR TITLE
Feat/components (part 2)

### PR DESCRIPTION
**Pull request by lyolya95**
2 - Styles and markup are implemented
3 - Used stateless/stateful approach
4 - Used event handlers
5 - Use lifecycle methods or useEffect hook

### CODE REVIEW

File: src/components/Movies/Filters/Filters.module.css [hunk 1]
```diff
@@ -50,6 +49,6 @@
 
 	letter-spacing: 0.888889px;
 	text-transform: uppercase;
-
+  cursor: pointer;
```
Comment (mmarkelov) [inline src/components/Movies/Filters/Filters.module.css:52]: missed tab

File: src/components/Movies/Movies.tsx [hunk 2]
```diff
@@ -1,20 +1,89 @@
...
+    isFlag: isShowDelete,
+    onOpen: openDelete,
+    onClose: closeDelete,
+  } = useToggle();
+
+  const {
+    isFlag: isShowEdit,
+    onOpen: openEdit,
+    onClose: closeEdit,
+  } = useToggle();
```
Comment (mmarkelov) [inline src/components/Movies/Movies.tsx:29]: It's ok, but seems a bit overheated. Maybe we can use this approach:
```js
const [openedModal, setOpenedModal] = useState<'event' | 'delete' | 'edit' | null>(null);
``` 

So we will need to change only the type of modal

File: src/utils/components/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React, { FC, useState } from "react";
+import { InputProps } from "./types";
+import styles from "./Input.module.css";
+
+export const Input: FC<InputProps> = ({
+  title,
+  placeholder,
+  value: initialValue,
+}) => {
+  const [value, setValue] = useState<string | undefined>(initialValue);
```
Comment (mmarkelov) [inline src/utils/components/Input/Input.tsx:10]:
```js
const [value, setValue] = useState(initialValue || '');
```

File: src/utils/components/Modal/Modal.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+              className={styles.close}
+              onClick={handleClose}
+            >
+              ×
+            </button>
+          </div>
+          <div className={styles.body}>
+            <p> {children}</p>
+          </div>
+          {footerContent ? footerContent : ""}
```
Comment (mmarkelov) [inline src/utils/components/Modal/Modal.tsx:41]:
```js
{footerContent || ""}
```

File: src/utils/components/Textarea/Textarea.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React, { FC, useState } from "react";
+import { TextareaProps } from "./types";
+import styles from "./Textarea.module.css";
+
+export const Textarea: FC<TextareaProps> = ({
+  title,
+  placeholder,
+  value: initialValue,
+}) => {
+  const [value, setValue] = useState<string | undefined>(initialValue);
```
Comment (mmarkelov) [inline src/utils/components/Textarea/Textarea.tsx:10]:
```js
const [value, setValue] = useState(initialValue || '');
```

File: src/components/Movies/Filters/Filters.module.css (hunk 1 above)
Comment (lyolya95) [reply src/components/Movies/Filters/Filters.module.css]: b84fe51ace38a9ef9c17ab2d06e3c61494e37c58

File: src/utils/components/Input/Input.tsx (hunk 3 above)
Comment (lyolya95) [reply src/utils/components/Input/Input.tsx]: b84fe51ace38a9ef9c17ab2d06e3c61494e37c58

File: src/utils/components/Modal/Modal.tsx (hunk 4 above)
Comment (lyolya95) [reply src/utils/components/Modal/Modal.tsx]: b84fe51ace38a9ef9c17ab2d06e3c61494e37c58

File: src/utils/components/Textarea/Textarea.tsx (hunk 5 above)
Comment (lyolya95) [reply src/utils/components/Textarea/Textarea.tsx]: b84fe51ace38a9ef9c17ab2d06e3c61494e37c58

File: src/components/Movies/Movies.tsx (hunk 2 above)
Comment (lyolya95) [reply src/components/Movies/Movies.tsx]: thank you i change it af6fc3753b3eab86e5a78f77298408a317539988

**Pull request merged**